docs/design-proposals/apiserver/openstack-api-support.md

```diff
@@ -0,0 +1,615 @@
+---
+title: Support Openstack Compute APIs in Arktos
+authors:
+  - "@yb01"
+---
+
+# Support Openstack Compute APIs in Arktos
```

Comment:
Can you add a TOC at the beginning? This document is very long and layout is not clear.

Reply:
good idea. added TOC